SPECS/openvswitch.spec

```diff
@@ -352,6 +353,11 @@ tunnels using IPsec.
 %systemd_postun openvswitch-ipsec.service
 
 %changelog
+* Fri Jun 21 2024 David Morel <david.morel@vates.tech> - 2.17.7-1.3
+- Link with xs-openssl instead of openssl
+- Tweak ovs-ctl script to pass ciphers and protocols ovsdb-server to command
+  line by default in order to disable weak ciphers
```

Comment:
I have difficulties understanding the wording here

Reply:
I rephrased it to try to make it clearer.